### PR TITLE
Feature: add customer stories

### PR DESCRIPTION
**Pull request by colmarius**
#### Description
- we add user stories index page (similar to webinars listing)
- we add the user story page (basic similar to a post, missing company section)
- add navigation to bottom of the user story: for now display max top three recent published user stories
- added support for multi languages
- `gatsby-node.js` generates: both Post and UserStory pages
- some shared React components extracted, reused between Post and UserStory pages

#### Todo
- [x] render company section on user story
- [x] card header has optional subtitle, used on user story
- [x] get style / overall feedback and address it 
- [x] test multi language user story page
- [ ] SEO checks with Armon

### CODE REVIEW

**Comment by colmarius** [comment]
There are now published user stories, probably that is why netlify preview deploy is failing.

File: src/components/customerStories/CompanyContent.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  faClock,
+  faMoneyBillAlt,
+  faUserFriends,
+  faIndustry,
+  faSeedling
+} from '@fortawesome/free-solid-svg-icons';
+
+import { LongContent } from '../LongContent';
+
+export const CompanyContent = ({ company }: {| company: Company |}) => (
```
Comment (ovcOS) [inline src/components/customerStories/CompanyContent.jsx]: Same as above. For someone new to the code, `CompanyContent` is pretty ambiguous

Comment (ovcOS) [reply src/components/customerStories/CompanyContent.jsx]: I see you named other files `CustomerStory<something>`, so maybe we can keep that pattern?

File: src/components/customerStories/CompanyContent.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        {company.employeeCount} Employees
+      </div>
+      <div className="row mx-auto py-3">
+        <FontAwesomeIcon fixedWidth size="lg" icon={faIndustry} title="Sector" className="mr-4" />
+        {company.sector}
+      </div>
+      <div className="row mx-auto py-3">
+        <FontAwesomeIcon fixedWidth size="lg" icon={faSeedling} title="Stage" className="mr-4" />
+        {company.stage}
+      </div>
```
[the remark this answers is not in the document]
Comment (colmarius) [reply src/components/customerStories/CompanyContent.jsx]: I was thinking about it, wanted to wait for changes to be done. Makes sense, I will do it.

**Comment by ovcOS** [comment]
Screen Shot 2020-01-29 at 11 11 02 AM

One more comment: the quote on this card feels too prominent. I'd make the text slightly smaller or the font-weight lighter, and I'd add a bit more padding between the logo and the quote

**Comment by colmarius** [comment]
I will work on restyling the quote too ;-).

File: src/components/Content.jsx [hunk 3]
```diff
@@ -28,6 +28,66 @@ export const ContentBody = ({ children }: {| children: Node | Array<Node> |}) =>
   </main>
 );
 
+export const CardLink = ({
```
[the remark this answers is not in the document]
Comment (colmarius) [reply src/components/Content.jsx]: makes sense Uri, let's follow the boy scout rule, to leave code slightly better than we found it :-).

File: src/components/LongContent.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import React from 'react';
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { MDXProvider } from '@mdx-js/react';
+
+import { Image, Lead } from '../components/Markdown';
+
+export const LongContent = ({
```
[the remark this answers is not in the document]
Comment (colmarius) [reply src/components/LongContent.jsx]: renamed LongContent to LongText to match the contentful field "long text".

File: src/components/Header.jsx [hunk 5]
```diff
@@ -8,20 +8,23 @@ import { calculatorUrl, targetBlank, trackSignupGoogleAnalytics } from '../helpe
...
+  title,
+  subtitle = '',
+  lang,
+  language
 }: {|
+  title: string,
+  subtitle?: string,
   lang: string,
-  data: {| contentfulPage: Page, site: { siteMetadata: { siteUrl: string } } |}
+  language: string
```
[the remark this answers is not in the document]
Comment (colmarius) [reply src/components/Header.jsx]: renamed, now we have `lang` and `documentLang`, they are needed to show the language hint: this page only available in “this” language.

File: src/components/customerStories/CompanySummary.jsx [hunk 6]
```diff
@@ -28,44 +36,18 @@ export const CompanySummary = ({ company }: {| company: Company |}) => (
...
+          <FontAwesomeIcon
+            fixedWidth
+            size="lg"
+            icon={icon}
+            title={iconTitle}
+            className="mr-4"
+          />
+          {description}
+        </div>
+      )}
```
Comment (ovcOS) [inline src/components/customerStories/CompanySummary.jsx]: 👍

**Pull request merged**